docs/configuration.md

```diff
@@ -1207,6 +1207,25 @@ Apart from these, the following properties are also available, and may be useful
 <tr>
   <td><code>spark.ui.view.acls</code></td>
   <td>Empty</td>
+</tr>
+<tr>
+  <td>spark.executor.multiPerWorker</td>
+  <td>false</td>
+  <td>
+    enable user to run multiple executors in the same worker.
+  </td>
+</tr>
+<tr>
+  <td>spark.executor.maxCoreNumPerExecutor</td>
+  <td>1</td>
+  <td>
+    set the max number of cores assigned to each executor; this property is only valid when
+    <code>spark.executor.multiPerWorker</code> is set to true.
+  </td>
+</tr>
+<tr>
+  <td>spark.executor.extraClassPath</td>
```

Comment:
wait, what is this doing here? The description doesn't match this config. Did you add this by mistake?